cli/tests/integrations/test_marathon_pod.py

```diff
@@ -220,7 +259,7 @@ def _pod(pod_id, file_path):
             _assert_pod_remove(pod_id, extra_args=['--force'])
 
 
-def _wait_for_instances(expected_instances, max_attempts=10):
+def _wait_for_instances(expected_instances, max_attempts=20):
```

Comment:
I had to increase the time because the `pod update` test now spins up six instances.